frontend/src/metabase/admin/permissions/components/PermissionsPageLayout/PermissionsPageLayout.tsx

```diff
@@ -47,7 +47,7 @@ type PermissionsPageLayoutProps = {
   confirmBar?: ReactNode;
   diff?: PermissionsGraph;
   isDirty?: boolean;
-  onSave?: () => void;
+  onSave: () => void;
```

Comment:
All the users of this component passed onSave, and it needs to be defined for where it's used

Reply:
unrelated to this PR?